packages/rollup-plugin/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Should we specify the files to published on npm here? This is what we do with the other packages (we publish only `dist` folder and type definitions).

Reply:
Just added a 'files' key like I saw here: https://github.com/honeybadger-io/honeybadger-js/blob/c6970426f1c38b6dd07c1d355fc20bd2730521c8/packages/js/package.json#L32